pkgs/by-name/ht/httpy-cli/package.nix

```diff
@@ -0,0 +1,30 @@
+{ lib
+, python3Packages
+, fetchPypi
+ }:
+
+python3Packages.buildPythonPackage rec {
+  pname = "httpy-cli";
+  version = "1.1.0";
+  format = "setuptools";
+
+  src = fetchPypi {
+    inherit pname version;
```

Comment:
Don't reuse `pname`, use a string literal instead.

Reply:
Is this what you asked for?
Because I just copied the examples from the documentation here https://nixos.wiki/wiki/Packaging/Python#install_binary_release

Reply:
We usually do not reuse the derivation attribute `pname`, however, we reuse the `version`.

Reply:
`repo = pname;` is nice for DRY but creates a binding that goes too far.

See further information about this here: https://github.com/nix-community/nixpkgs-lint/issues/21